android/assets/jsons/translations/Russian.properties

```diff
@@ -624,7 +622,7 @@ Convert production to science at a rate of [rate] to 1 = Обмен произв
 The city will not produce anything. = Город ничего не будет производить.
 Worked by [cityName] = Обрабатывается в г. [cityName]
  # Requires translation!
-Lock = 
+Lock = Блокировать
```

Comment:
I'd like to have more context on this - is it a noun (like a physical lock with a key), a verb (like a "lock city production" or something alike) or something else? Currently translated as a second variant - verb.

Reply:
Yup, it's a verb (lock the tile)